packages/cosmic-proto/MAINTAINERS.md

```diff
@@ -8,7 +8,7 @@ The generated code is determined by the contents of `protos` and the config of `
 
 ## Maintaining protos
 
-The `protos` are held in this source tree and updated from `golang/cosmos/proto` during `yarn codegen` by `yarn protos-update`. The `cosmos` protos were sourced from [@protobufjs/cosmos](https://www.npmjs.com/package/@protobufs/cosmos) per [Telescope's instructions](https://github.com/cosmology-tech/telescope?tab=readme-ov-file#add-protobufs). However we don't use that as a dependency of the package because we need a more manual approach to merge with the Golang-managed protos in the repo.
+The `protos` are held in this source tree and updated from `golang/cosmos/proto` during `yarn codegen` by the `update-protos.sh` script.
```

Comment:
This is ambiguous whether `yarn codegen` invokes `update-protos.sh` or vice-versa. Make it clear that you call `yarn codegen` and consider letting folks just read the implementation if they want to see how it works.

I've updated the instructions on the [Upgrading the Interchain Stack](https://github.com/Agoric/agoric-sdk/wiki/Upgrading-the-Interchain-Stack) wiki page.